### ISSUE TITLE
Find map of percent of Spanish-speaking people by Congressional district

### ISSUE THREAD

**Issue by Frijol** [edgi-govdata-archiving/EEW_Planning#388]

**Comment by ericnost** [comment]
https://data.census.gov/cedsci/map?q=United%20States&t=Language%20Spoken%20at%20Home&g=0100000US.50016&tid=ACSST1Y2019.S1601&hidePreview=false&vintage=2019&layer=VT_2019_500_16_PY_D1&cid=S1601_C02_004E&palette=Teal&break=5&classification=Natural%20Breaks&mode=customize

https://edgi.slack.com/archives/CKN6FMX42/p1612296729005900

Or do we want to be more specific i.e. an embeddable map? As noted, I'd be happy to download the Census data and craft a static map for the website?

**Comment by Frijol** [comment]
@lesliealanis?
